pkg/services/ngalert/schedule/fetcher.go

```diff
@@ -7,14 +7,14 @@ import (
 	"github.com/grafana/grafana/pkg/services/ngalert/models"
```

Comment:
Not for this PR, but: Does this really need its own tiny file?

Reply:
I think the only reason for that file is to separate data access logic. I do not have a strong opinion on where it should be.